.dockerignore

```
@@ -0,0 +1,11 @@
+.env*
+.git*
+
+**/.github
+**/.venv
+**/.vscode
+**/ansible
+
+*.md
+
+__pycache__
```

Bonus things we might consider, including that we want to ignore all nested `__pycache__` folders too (note that the `**` is important in this file, unlike in `gitignore` which assumes it by default)

```suggestion
**/__pycache__
.pytest_cache
```